sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -402,15 +397,13 @@ export class ServiceBusManagementClient extends ServiceClient {
     let options: ServiceBusManagementClientOptions;
     let fullyQualifiedNamespace: string;
     let credentials: SasServiceClientCredentials | TokenCredential;
```

Comment:
Now that you have the private member, you dont need this variable anymore

Reply:
No, I'm calling this `super(credentials, serviceClientOptions);` after the if-else and I'm getting the credentials inside the if-else and hence I'll need the variable.